.gitconfig

```diff
@@ -0,0 +1,5 @@
+[user]
+	email = hdm23061993@gmail.com
```

Comment:
Are you sure this file should be included into repo?

Reply:
Fixed, sorry for the mess